spring-cloud-services-spring-connector/src/test/java/io/pivotal/spring/cloud/service/config/CredHubPropertyMaskingContextInitializerIntegrationTests.java

```diff
@@ -0,0 +1,100 @@
+package io.pivotal.spring.cloud.service.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.env.EnvironmentEndpoint;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootContextLoader;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.core.env.CompositePropertySource;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.core.env.StandardEnvironment;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CredHubPropertyMaskingContextInitializerIntegrationTests {
```

Comment:
This file is almost identical to VaultPropertyMaskingContextInitializerIntegrationTests.java. Is it possible to abstract the shared behavior out so we don’t need to remember to keep them in sync?

Reply:
United the two test classes with the newest commit.